cmd/nebraska/metrics.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	defaultMetricsUpdateInterval = time.Second * time.Duration(5)
```

Comment:
This should be fine too:

```suggestion
	defaultMetricsUpdateInterval = 5 * time.Second
```